views/pages/resources/cheque-requests.pug

```diff
@@ -0,0 +1,38 @@
+extends /views/templates/frontend-base.pug
+
+append styles
+    link(href="/assets/css/pages/resources/cheque-requests.css" rel="stylesheet")
+append scripts
+    script(src="/assets/js/collapsible.js" defer)
+
+block body
+    +banner-image("/assets/img/banners/mathsoc-wall.jpeg")

```

Comment:
Thoughts on whether this page would look better without a banner image at the top?

Reply:
Agreed, done